catchup/universalFetcher_test.go

```diff
@@ -127,3 +132,150 @@ func TestUGetBlockUnsupported(t *testing.T) {
 	require.Nil(t, cert)
 	require.Equal(t, int64(duration), int64(0))
 }
+
+// TestprocessBlockBytesErrors checks the error handling in processBlockBytes
+func TestProcessBlockBytesErrors(t *testing.T) {
+
+	blk := bookkeeping.Block{
+		BlockHeader: bookkeeping.BlockHeader{
+			Round: basics.Round(22),
+		},
+	}
+
+	blkData := protocol.Encode(&blk)
+	bc := protocol.EncodeReflect(rpcs.PreEncodedBlockCert{
+		Block: blkData,
+	})
+
+	// Check for cert error
+	_, _, err := processBlockBytes(bc, 22, "test")
+	var wcfpe wrongCertFromPeerError
+	require.True(t, errors.As(err, &wcfpe))
+	err.Error()
```

Comment:
why ?

Reply:
Replaced this with a proper test to test the error types.